js-exercises/deep-copy-object/deepCopyObject.js

```diff
@@ -0,0 +1,24 @@
+const deepCopyObject = value => {
+  if (isPrimitive(value)) {
+    return value;
+  }
+
+  return deepCopyOfObject(value);
+};
+
+function deepCopyOfObject(object) {
```

Comment:
'object' is a reserved keyword. It would be better to name inputObject etc.

Reply:
Agreed.